VMEncryption/main/handle.py

```diff
@@ -1261,15 +1261,60 @@ def decrypt_inplace_with_separate_header_file(passphrase_file,
                                      status_prefix,
                                      ongoing_item_config)
 
-def enable_encryption_all_in_place(passphrase_file, encryption_marker, disk_util, bek_util):
+
+def enable_encryption_all_format(passphrase_file, encryption_marker, disk_util, bek_util):
     """
-    if return None for the success case, or return the device item which failed.
+    In case of success return None, otherwise return the device item which failed.
     """
-    logger.log(msg="executing the enableencryption_all_inplace command.")
+    logger.log(msg="executing the enable_encryption_all_format command")
+
+    device_items = find_all_devices_to_encrypt(encryption_marker, disk_util, bek_util)
+    # Don't encrypt partitions that are not even mounted
+    device_items_to_encrypt = filter(lambda di: di.mount_point is not None and di.mount_point != "", device_items)
+
+    msg = 'Encrypting and formatting {0} data volumes'.format(len(device_items_to_encrypt))
+    logger.log(msg);
+
+    hutil.do_status_report(operation='EnableEncryptionAllFormat',
```

Comment:
Inconsistent naming - other places in code refer to 'EnableEncryptionFormatAll' where "All" is the suffix